src/modality-agnostic-files.md

```diff
@@ -429,6 +432,25 @@ values in the corresponding column is a transformation of values from other
 columns (for example a summary score based on a subset of items in a
 questionnaire).
 
+## Demographics file
+
+Template:
+
+```Text
+phenotype/
+    demographics.tsv
+    demographics.json
+```
+
+The demographics file is an OPTIONAL tabular phenotypic file in
+the `phenotype/` directory meant to house common subject demographics.
+For example demographics like gender, race, and household income.
+A demographics file is RECOMMENDED to use when any participant has
+more than one session of any type.
+It does not replace the participants file, which is meant for data about
+each participant at first visit. It instead supplements the participants file
```

Comment:
```suggestion
each participant at first session. It instead supplements the participants file
```
unless you intentionally call this "visit", I'd stick with "session". We have had (very) long debates  in our group about the difference between the two ...